packages/mdc-menu/simple/mdc-simple-menu.scss

```diff
@@ -189,6 +190,23 @@ $mdc-simple-menu-item-fade-duration: .3s;
     opacity: .18;
   }
 
+  .mdc-list-item[aria-disabled="true"] {
+    cursor: default;
```

Comment:
have we considered putting `pointer-events: none` on here? That may also take care of not having to check in event handling logic whether the component is disabled.